timemachine/lib/potentials.py

```diff
@@ -34,6 +36,27 @@ def bound_impl(self, precision):
         return custom_ops.BoundPotential(self.unbound_impl(precision), self.params)
 
 
+class SummedPotential:
```

Comment:
Should this inherit from the wrapper?

Reply:
I went back and forth on this; the method signature is the same as `CustomOpWrapper`, but the constructor and nontrivial parts of the implementation are different. I also thought about adding an abstract base class from which both inherit, but opted to keep it as simple as possible. I'll reconsider extending `CustomOpWrapper`

Reply:
On further reflection, I think it does make sense to inherit from the wrapper, and simplifies things quite a bit! Changed in 4a26f7c.